Code/GraphMol/FMCS/testFMCS_Unit.cpp

```diff
@@ -705,6 +707,50 @@ void testJSONParameters() {
     BOOST_LOG(rdInfoLog) << "\tdone" << std::endl;
 }
 
+void testAtomCompareDistance() {
+
+    BOOST_LOG(rdInfoLog) << "\n-------------------------------------" << std::endl;
+    BOOST_LOG(rdInfoLog) << "Testing Atom Compare Distance Cutoff" << std::endl;
+
+    std::string s1="CCCCC";
+    ROMOL_SPTR mol1(SmilesToMol(s1));
+    ROMOL_SPTR mol2(SmilesToMol(s1));
+
+    DGeomHelpers::EmbedMolecule(*mol1);
+    Conformer &conf1 = mol1->getConformer();
+
+    Conformer second(conf1);
+    Conformer &conf2 = second;
+    conf2.setAtomPos(0, RDGeom::Point3D(0,0,0));
+    mol2->addConformer(&conf2);
```

Comment:
Well stupid me - it was due to a double free. The second conformer was being called twice - once when it leaves the scope, and once the owning Mol destroys it via its own shared_ptr.